src/translations/pt.json

```diff
@@ -1,55 +1,56 @@
-    "customer_master.first_name.label": "First Name",
-    "customer_master.last_name.label": "Last Name",
-    "customer_master.address_1.label": "Address 1",
-    "customer_master.address_2.label": "Address 2",
-    "customer_master.address_3.label": "Address 3",
-    "customer_master.city.label": "City",
-    "customer_master.state.label": "State",
-    "customer_master.country.label": "Country",
-    "customer_master.email.label": "Email",
-    "customer_master.phone.label": "Phone Number",
-    "customer_master.gps.label": "GPS Coordinates",
-
-    "customer_type": "Customer Type",
-    "customer_type.header": "Customer Type",
-    "customer_type.column.id.label": "ID",
-    "customer_type.column.name": "Customer Name",
-    "customer_type.name.label": "Customer Type Name",
-
-    "driver": "Driver Master",
-    "driver_master.header": "Driver Master",
-    "driver_master.column.id.label": "ID",
-    "driver_master.column.transport_name.label": "Transport Name",
-    "driver_master.transporter.label": "Transporter",
-    "driver_master.name.label": "Driver Name",
-    "driver_master.epod_username.label": "ePOD Username",
-    "driver_master.epod_password.label": "ePOD Password",
-    "driver_master.epod_username.placeholder": "Username",
-    "driver_master.epod_password.placeholder": "Password",
+     "cost": "Cost form",
+     "cost_form.header": "Cost form",
+     
+     "customer": "Customer Master",
+     "customer_master.header": "Customer Master",
+     "customer_master.customer_type_name.label": "Customer Type Name",
+     "customer_master.name.label": "Name",
+     "customer_master.first_name.label": "First Name",
+     "customer_master.last_name.label": "Last Name",
+     "customer_master.address_1.label": "Address 1",
+     "customer_master.address_2.label": "Address 2",
+     "customer_master.address_3.label": "Address 3",
+     "customer_master.city.label": "City",
+     "customer_master.state.label": "State",
+     "customer_master.country.label": "Country",
+     "customer_master.email.label": "Email",
+     "customer_master.phone.label": "Phone Number",
+     "customer_master.gps.label": "GPS Coordinates",
+     
+     "customer_type": "Customer Type",
+     "customer_type.header": "Customer Type",
+     "customer_type.column.id.label": "ID",
+     "customer_type.column.name": "Customer Name",
+     "customer_type.name.label": "Customer Type Name",
+     
+     "driver": "Driver Master",
+     "driver_master.header": "Driver Master",
+     "driver_master.column.id.label": "ID",
+     "driver_master.column.transport_name.label": "Transport Name",
+     "driver_master.transporter.label": "Transporter",
+     "driver_master.name.label": "Driver Name",
+     "driver_master.epod_username.label": "ePOD Username",
+     "driver_master.epod_password.label": "ePOD Password",
+     "driver_master.epod_username.placeholder": "Username",
+     "driver_master.epod_password.placeholder": "Password",
```

Comment:
indentation issue - look on what you added and the older code, there is small difference.